docs/modules/ROOT/pages/microprofile-certification/5.36.0/microprofile-4.1-tck-results-full-5.36.0.adoc

```diff
@@ -0,0 +1,165 @@
+[[tck-results]]
+= TCK Results
+
+As required by the
+https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License],
+following is a summary of the TCK results for releases of MicroProfile Config 2.0.
```

Comment:
`summary of the TCK results for releases of MicroProfile Config 2.0.`

But is summary for MicroProfile 4.1, and not just Config.

Reply:
```suggestion
following is a summary of the TCK results for releases of MicroProfile 4.1 Specifications.
```